stablehlo/dialect/Base.td

```diff
@@ -146,6 +187,9 @@ def HLO_PredOrIntTensor : TensorOf<[HLO_Pred, HLO_Int]>;
 // Any floating-point or complex tensor types
 def HLO_FpOrComplexTensor : TensorOf<[HLO_Float, HLO_Complex]>;
 
+// Any floating-point or quantized tensor types
+def HLO_FpOrQuantizedIntTensor : TensorOf<[HLO_Float, HLO_QuantizedInt]>;
```

Comment:
I asked this in the T2x PR, but how many FpTensor ops also take quantized types? Don't need exact number, but if the answer is 90% then lets update FpTensor to allow per-tensor quantized ints

Reply:
8 out of 14 FpTensor OPs take quantized types. Better to keeping Quantized type separate?

Reply:
revisited: there are 6 ops which doesn't take quantized types, few also take other types like complex, it appear ok to keep `float` and `quant` separate and use `HLO_FpOrQuantizedIntTensor `.